packages/core/src/components/textarea/test/default/textarea.e2e.ts

```diff
@@ -0,0 +1,42 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/textarea/test/default/index.html';
+
+test.describe('tds-textarea', () => {
+  test('renders default textarea correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tdsTextarea = page.locator('tds-textarea');
+    expect(tdsTextarea).toBeTruthy();
+    /* Expect no difference in screenshot  */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('default textarea - click on text area - cursor should appear', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const textarea = page.locator('tds-textarea').locator('textarea');
+    const myEventSpy = await page.spyOnEvent('click');
+    await textarea.click();
+    /* Expect to have received an event from clicking on the textarea */
+    expect(myEventSpy).toHaveReceivedEvent();
+    const textareaCursorState = await textarea.evaluate(
+      (button2) => getComputedStyle(button2).cursor,
```

Comment:
Is there a benefit to renaming "button2" to something else in this context?

Reply:
hmm totally, updated to style instead. But yeah `button2` was a copy pasta :D